addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -585,10 +585,9 @@ def fill_account_move_reversed_entry_id(env):
         env.cr, """
         UPDATE account_move am
         SET reversed_entry_id = am2.id
-        FROM account_invoice ai
-        JOIN account_invoice ai2 ON ai.refund_invoice_id = ai2.id
-        JOIN account_move am2 ON am2.old_invoice_id = ai2.id
-        WHERE am.reversed_entry_id IS NULL AND am.old_invoice_id = ai.id"""
+        FROM account_move am2
+        WHERE am.reversed_entry_id IS NULL AND am2.reverse_entry_id = am.id
```

Comment:
`reverse_entry_id` is not filled in this case AFAIK, so you should go to invoice table for this information.

Reply:
@pedrobaeza I don't understand your remark. in 13.0, `reverse_entry_id` became `reversed_entry_id` with a reversed relation. This is what this update does. Since reverse_entry_id did exist in 12, we can assume it was populated no?

Regarding the invoices, the update before this PR did not work, and you suggested in https://github.com/OCA/OpenUpgrade/issues/4376#issuecomment-2053617507 that there was nothing to do.

In any case I can confirm this PR works for me and correctly set the Reversal Entry field.

Reply:
But in v12, invoices were another thing, and this is filling the reverse link between invoices through the `refund_invoice_id` field. I don't think the `reverse_entry_id` was filled for the journal entries generated from the invoices that are refunded. Or am I wrong? Maybe both operations should be done.

Reply:
Ok, I updated to preserve the existing udpate but changing reverse_entry_id, then let the second update reverse the relation.

I can't test this as the database I'm working with does not have refund_invoice_id for some reason.

Reply:
Hello just in case this might help you to inspect or remember what Odoo SA did for v13 commit by commit: https://github.com/akretion/odoo-module-diff-analysis/tree/main/13.0/account

Reply:
Thanks Raphael. Well in this case, the relevant commit is imp-ref-accounting-pocalypse-yeaaahh, so... yeah. :)

Reply:
But your analysis tool is definitely super useful!

Reply:
Once I'll have cleaned up it a bit more I'll propose to integrate these key change commits along the OpenUgrade analysis files. Then it will be up to the OpenUpgrade leaders to accept it inside the repo or not...